packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.service.js

```diff
@@ -12,39 +12,57 @@ import Flavor from './flavor.class';
 import FlavorGroup from './flavor-group.class';
 
 import {
-  CPU_FREQUENCY,
   CATEGORIES,
+  CPU_FREQUENCY,
+  DEFAULT_CATALOG_ENDPOINT,
   LEGACY_FLAVORS,
 } from './flavors-list.constants';
 
 export default class FlavorsList {
   /* @ngInject */
   constructor(
     $q,
+    $http,
     coreConfig,
     CucPriceHelper,
     OvhApiCloudProjectFlavor,
     OvhApiOrderCatalogPublic,
   ) {
     this.$q = $q;
+    this.$http = $http;
     this.coreConfig = coreConfig;
     this.CucPriceHelper = CucPriceHelper;
     this.OvhApiCloudProjectFlavor = OvhApiCloudProjectFlavor;
     this.OvhApiOrderCatalogPublic = OvhApiOrderCatalogPublic;
   }
 
-  getFlavors(serviceName, currentRegion) {
+  getCatalog(endpoint, ovhSubsidiary) {
```

Comment:
Yep, there are duplicate usage and it could be good to use one.
As you know it was there before and i prefer to let it like this because we are bit late for QC.